code/stable/gamephysics/SRS/GamePhysics_SRS.tex

```diff
@@ -510,16 +510,19 @@ \subsubsection{General Definitions}
         
 \\ \midrule \\
 Equation & \begin{displaymath}
-           \mathbf{g}=\frac{G M}{\text{||}\mathbf{r}\text{||}^{2}} \mathbf{\hat{r}}
+           \mathbf{g}=-\frac{G M}{\mathbf{r}^{2}} \mathbf{\hat{r}}
            \end{displaymath}
 \\ \midrule \\
 Description & \begin{symbDescription}
               \item{$\mathbf{g}$ is the gravitational acceleration ($\frac{\text{m}}{\text{s}^{2}}$)}
               \item{$G$ is the gravitational constant ($\frac{\text{m}^{3}}{(\text{kg}\text{s}^{2})}$)}
               \item{$M$ is the mass of the larger rigid body (kg)}
-              \item{$\text{||}\mathbf{r}\text{||}$ is the Euclidean norm of the displacement (m)}
-              \item{$\mathbf{\hat{r}}$ is the displacement unit vector (m)}
+              \item{$\mathbf{r}$ is the displacement (m)}
+              \item{$\mathbf{\hat{r}}$ is the unit vector directed from the large mass to the smaller mass (m)}
```

Comment:
The text should say from the centre of the large mass to the centre of the smaller mass.

Reply:
I used the definition of r from the wiki source as is, but I have updated as per your request.

Reply:
@oluowoj, once you push this change to the definition of r, I can approve the pull request from the point of view of stable.

Reply:
@smiths, I pushed  the updated definition, build is still failing though